fly.toml

```diff
@@ -0,0 +1,36 @@
+# fly.toml app configuration file generated for comapeo-cloud on 2024-10-07T20:59:21+01:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = 'comapeo-cloud'
+primary_region = 'ord'
```

Comment:
First of all, thank you for thinking of Chicago.

Second, given that we plan to demo this in Colombia, should we set this to `bog` (Bogotá) instead?

Reply:
changed to iad as default, we can set to bog when deploying for a specific use